### PR TITLE
fixing tracing stuff

### PR DESCRIPTION
**Pull request by paxtonhare**

### CODE REVIEW

File: marklogic-data-hub/src/server-side/impl/trace-lib.xqy [hunk 1]
```diff
@@ -610,7 +610,7 @@ declare function trace:get-traces($page as xs:int, $page-length as xs:int)
   let $end := $start + $page-length - 1
   let $count := xdmp:estimate(/trace)
   let $traces :=
-    for $trace in cts:search(/trace, cts:true-query(), ("unfiltered", cts:index-order(cts:element-reference(xs:QName("created")), "descending")))[$start to $end]
+    for $trace in cts:search(/trace, cts:true-query(), ("unfiltered", cts:index-order(cts:path-reference("/trace/created"), "descending")))[$start to $end]
```
Comment (dmcassel) [inline marklogic-data-hub/src/server-side/impl/trace-lib.xqy]: has the traceDB config been updated to add this path range index?

File: marklogic-data-hub/src/server-side/impl/trace-lib.xqy [hunk 2]
```diff
@@ -628,8 +628,7 @@ declare function trace:get-trace($id as xs:string)
 {
   let $query :=
     cts:or-query((
-      cts:element-range-query(xs:QName("traceId"), "=", $id, ("collation=http://marklogic.com/collation/codepoint")),
-      cts:json-property-range-query("traceId", "=", $id, ("collation=http://marklogic.com/collation/codepoint"))
+      cts:path-range-query("/trace/traceId", "=", $id, ("collation=http://marklogic.com/collation/codepoint"))
```
Comment (dmcassel) [inline marklogic-data-hub/src/server-side/impl/trace-lib.xqy]: has the traceDB config been updated to add this path range index?

File: marklogic-data-hub/src/server-side/impl/trace-lib.xqy [hunk 3]
```diff
@@ -642,12 +641,11 @@ declare function trace:get-traceIds($q as xs:string?)
       cts:element-value-query(xs:QName("identifier"), fn:lower-case($q) || "*", "wildcarded")
     else ()
   let $results :=
-    cts:element-value-co-occurrences(
-      xs:QName("traceId"),
-      xs:QName("identifier"),
+    cts:value-co-occurrences(
+      cts:path-reference("/trace/traceId"),
+      cts:path-reference("/trace/identifier"),
```
Comment (dmcassel) [inline marklogic-data-hub/src/server-side/impl/trace-lib.xqy]: has the traceDB config been updated to add this path range index?

File: marklogic-data-hub/src/server-side/impl/trace-lib.xqy (hunk 1 above)
Comment (paxtonhare) [reply marklogic-data-hub/src/server-side/impl/trace-lib.xqy]: it never had a range index. dunno how it ever worked.

File: marklogic-data-hub/src/server-side/impl/trace-lib.xqy (hunk 2 above)
Comment (paxtonhare) [reply marklogic-data-hub/src/server-side/impl/trace-lib.xqy]: never had this other index.

File: marklogic-data-hub/src/server-side/impl/trace-lib.xqy (hunk 3 above)
Comment (paxtonhare) [reply marklogic-data-hub/src/server-side/impl/trace-lib.xqy]: ditto

**Pull request merged**